packages/shared/src/consts.ts

```diff
@@ -40,19 +40,27 @@ const MINIMUM_SWAP_AMOUNTS: Partial<
 export const getMinimumSwapAmount = (network: ChainflipNetwork, asset: Asset) =>
   MINIMUM_SWAP_AMOUNTS[network]?.[asset] ?? '0';
 
+// https://developers.circle.com/developer/docs/usdc-on-testnet#usdc-on-ethereum-goerli
+const GOERLI_USDC_CONTRACT_ADDRESS =
+  '0x07865c6E87B9F70255377e024ace6630C1Eaa37F';
+
 export const ADDRESSES = {
+  [ChainflipNetworks.backspin]: {
+    FLIP_CONTRACT_ADDRESS: '0x10C6E9530F1C1AF873a391030a1D9E8ed0630D26',
+    USDC_CONTRACT_ADDRESS: '0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0',
+    VAULT_CONTRACT_ADDRESS: '0xB7A5bd0345EF1Cc5E66bf61BdeC17D2461fBd968',
+    STATE_CHAIN_GATEWAY_ADDRESS: '0xeEBe00Ac0756308ac4AaBfD76c05c4F3088B8883',
+  },
```

Comment:
are these contract addresses the same on backspin everytime?

Reply:
yes i think so, at least the bouncer will use this addresses in the `reset-backspin` workflow in https://github.com/chainflip-io/chainflip-testnet-tools